ndcube/ndcube_sequence.py

```diff
@@ -267,7 +286,7 @@ def crop(self, *points, wcses=None):
         item = self._get_sequence_crop_item(*points, wcses=wcses)
         return self[item]
 
-    def crop_by_values(self, lower_corner, upper_corner, units=None, wcses=None):
+    def crop_by_values(self, *points, units=None, wcses=None):
```

Comment:
While I think this change is how the API should be, is it actually a subtle API change that will need a 3.0 release?

Reply:
I think this hopefully fixes the API, I have added a unit test for this.